easybuild/easyconfigs/r/rtg-tools/rtg-tools-3.9.1-Java-1.8.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'CmdCp'
+
+name = 'rtg-tools'
+version = '3.9.1'
+versionsuffix = '-Java-%(javaver)s'
+
+homepage = 'https://www.realtimegenomics.com/products/rtg-tools'
+description = """ RTG Tools contains utilities to easily manipulate and accurately compare multiple VCF files,
+ as well as utilities for processing other common NGS data formats. """
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['https://github.com/RealTimeGenomics/rtg-tools/archive']
+sources = ['%(version)s.tar.gz']
+checksums = ['e4738c236a5a1ad13754500b2078c9500d66051ac305abf96730015230c4f830']
+
+builddependencies = [('ant', '1.10.5', versionsuffix, True)]
+dependencies = [('Java', '1.8', '', True)]
```

Comment:
@smoors You can simplify this a bit, since the `dummy` toolchain is used (so we don't need the `True`):

```python
builddependencies = [('ant', '1.10.5', versionsuffix)]
dependencies = [('Java', '1.8')]
```